paritytech/Nomidot
# feat: Separate preimage from proposal

closes https://github.com/paritytech/Nomidot/issues/97

preimage an proposal can be created in any order, they will end up being connected.
To test, connect to a local node and either:

- create a proposal with a preimage hash e.g `0x1c0a116889e9584b6709bf01235ece86c285597801c40826dbccd67f187739fe` for set_validator => this should create the proposal but no preimage
- create the preimage (`staking.setValidatorCount`: 150 to get the same preimage hash) => this should create the preimage and link it to the proposal

or the other way around:
- create the preimage (`staking.setValidatorCount`: 150 to get the same preimage hash) => this should create the preimage but there's no proposal
- create a proposal with the previously created preimage hash e.g `0x1c0a116889e9584b6709bf01235ece86c285597801c40826dbccd67f187739fe` for set_validator => this should create the proposal and link it to the preimage.
